apps/web/nuxt.config.ts

```diff
@@ -138,6 +138,7 @@ export default defineNuxtConfig({
   },
   viewport: {
     breakpoints: {
+      xs: 380,
       sm: 640,
       md: 640,
```

Comment:
**question(non-blocking)**: I didn't notice this before, but why do we have the same breakpoints for `sm` and `md`?

Do you know @maxiroellplenty?